src/mastra/workflows/logCoreAnalyzer/constants.ts

```diff
@@ -2,9 +2,9 @@
  * Input source types
  */
 export enum SOURCE_TYPE {
```

Comment:
Enum name should be standardized too
```suggestion
export enum SourceType {
```